Modules/Sources/Networking/Mapper/MediaListMapper.swift

```diff
@@ -1,3 +1,5 @@
+import Foundation
+
```

Comment:
As discussed in earlier PRs, when we go from framework to package we lose the Objective-C header file with the customary `#import <Foundation/Foundation.h>`. This requires files that access foundation types to explicitly import it.